recipes/catalystcoop.dbfread/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "catalystcoop.dbfread" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e79dc93ec2e5e4dc4f9a8daac2edc5b411026301fc80bd6ceb41b453d9c7ea36
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True # [py<36]
```

Comment:
If this is `noarch: python` remove the `skip` line. The limits down under `requirements` is enough to restrict the versions correctly.